inference-engine/tests/functional/inference_engine/ngraph_reader/fake_quantize_tests.cpp

```diff
@@ -254,9 +254,5 @@ TEST_F(NGraphReaderTests, ReadFQNetwork) {
     CommonTestUtils::fill_data(weights->buffer().as<float *>(), weights->size() / sizeof(float));
 
     Core reader;
-    auto nGraph = reader.ReadNetwork(model, weights);
-    CNNNetwork cnn(nGraph);
-    IE_SUPPRESS_DEPRECATED_START
-    cnn.begin();
-    IE_SUPPRESS_DEPRECATED_END
+    auto cnn = reader.ReadNetwork(model, weights);
```

Comment:
I don't know the historical reason why this test doesn't check v10 with v7, but as a minimum this test had a check that v10 with FQ can be converted to legacy representation. At the current moment conversion to legacy was removed. Please revert.

@GlebKazantaev Why don't we use compareCNNNetwork in this case? Also looks like `modelV5` isn't used in this test.

Reply:
reverted back